### DISCUSSION TITLE
有没有可能实现对比模式，就是一个问题多个模型进行回答

### DISCUSSION THREAD

**Discussion by wensonsmith** [lobehub/lobehub#1716] [Ideas]
经常会再多个模型中对比测试哪个效果会好一些，所以需要一直把问题在多个模型中复制粘贴。
在聊天窗口，时候可以多选模型，然后返回多个模型的值。

或者聊天窗口可以 split view ，类似于 https://chat.lmsys.org/ 这里的模式。

**Comment by Godoword** [comment]
我也有类似的需求，经常会对比不同模型。

**Comment by arvinxx** [comment]
有计划做的

**Comment by wensonsmith** [comment]
<img width="1144" alt="image" src="https://github.com/lobehub/lobe-chat/assets/2544185/aa7e1e17-3a2c-422f-bbed-b9a52d8eb30f">

OpenRouter 的这个 Playground 感觉很爽。 多个模型基本上是一起回答，可以作为参考。

https://openrouter.ai/playground

**Reply by arvinxx** [reply]
他这个咋多轮？交互上我不会这么做，这种平铺的话多轮要爆炸

**Reply by wensonsmith** [reply]
默认是折叠状态，就是输出的每一个的高度默认都是这样。需要点开查看。可以去体验一下，还是挺丝滑的。

不过多窗口 split view， 像 vs code 那样垂直分割感觉体验会好些。

**Comment by hlc1209** [comment]
+1 类似功能会很有用

**Comment by wensonsmith** [comment]
这个 feature 现在有排期了么 😢

**Reply by arvinxx** [reply]
还没有，估计要等8月中下了吧。最近在做文件上传/知识库

**Reply by wensonsmith** [reply]
> 还没有，估计要等8月中下了吧。最近在做文件上传/知识库

文件上传知识库都上线了，这个是不是得排上日程啦！ 🫶

**Reply by arvinxx** [reply]
是的，下个月会开始做多模型对比对话和分支对话，敬请期待~

**Reply by SoulGolf** [reply]
多模型对比排上日程了嘛 👀

**Reply by arvinxx** [reply]
暂时还没，准备排到明年 1 月或 2 月了。12月准备重构 Provider 的实现和交互，支持其他模态的模型选择和配置，以及迁移 db 到 pglite，进而实现 client db 和 server db 对齐。

**Comment by wensonsmith** [comment]
漫长的等待 😢

**Reply by arvinxx** [reply]
😂我们会加快脚步的

**Comment by rrrrrzy** [comment]
新的一年, 期待ing🥹...

**Comment by coder2z** [comment]
新的一年, 期待ing🥹...

**Reply by arvinxx** [reply]
3月给搞出来😆

**Comment by wensonsmith** [comment]
好久没关注了，这个有了么？

**Reply by leesmith288** [reply]
这个功能很棒诶，就像openrouter那样，目前还没有其它软件有这个功能

**Reply by wensonsmith** [reply]
> 这个功能很棒诶，就像openrouter那样，目前还没有其它软件有这个功能
 
Cherry Studio 很早就可以了，已经从 Lobechat 切到 Cherry Studio 了

**Comment by htdohk** [comment]
继续期待ing

**Comment by hedeqiang** [comment]
火山引擎也有一个模型对比，看起来也OK

**Reply by arvinxx** [reply]
我去看一眼

**Comment by SoulGolf** [comment]
继续期待ing

**Comment by rivertwilight** [comment]
现在，你可以试试通过群聊来实现这个需求。试试拉多个助手，设置不同模型，然后这样说：「你们同时来回答xx问题。」

➡️ [测试版](https://lobe-chat-database-git-feat-group-chat-lobehub-community.vercel.app/)

欢迎反馈。也许不是最好的解，但是希望能满足你的要求。

@SAnBlog @htdohk @wensonsmith @hlc1209

**Reply by m0eak** [reply]
> 现在，你可以试试通过群聊来实现这个需求。试试拉多个助手，设置不同模型，然后这样说：「你们同时来回答xx问题。」
> 
> ➡️ [测试版](https://lobe-chat-database-git-feat-group-chat-lobehub-community.vercel.app/)
> 
> 欢迎反馈。也许不是最好的解，但是希望能满足你的要求。
> 
> @SAnBlog @htdohk @wensonsmith @hlc1209

什么时候能上线啊，社区版可以使用吗

**Reply by rivertwilight** [reply]
预计今年秋天会上线。社区版和 Cloud 均可以使用。相关 RFC：[RFC 130](https://github.com/lobehub/lobe-chat/discussions/8920)

**Reply by SoulGolf** [reply]
> RFC 130

期待一年多了

**Reply by wensonsmith** [reply]
终于有了，可已经不重要了，我早切到 Cherry Studio 了。

作为用了 LobeChat 很久，还安利给朋友的前用户，对于这个功能来说，来的太晚了。

其次是设计上不够克制，让我萌生放弃 Lobechat 的第一个原因是这个：

<img width="3802" height="1728" alt="image" src="https://github.com/user-attachments/assets/08e43ff7-bfc4-4855-ab21-d40881869743" />

这些东西真的有人用么？ 反正我只想要一个空白的聊天窗口。加了这些东西多少还影响了加载速度。

Anyway, 还是感谢各位的付出！

**Reply by SoulGolf** [reply]
> 让我萌生放弃 Lobechat 的第一个原因是这个：

赞同，首页本应该是简洁的，首页这些内容让我产生不想使用随便聊聊的功能，但是每次都要手动切换到别的助手也是比较难受，设计上不够简洁
